media/transcoder/transcoder_test.go

```diff
@@ -180,6 +198,31 @@ func writeTestGCSFile(t *testing.T, dstBucket string, srcBucket string, srcObjec
 	}
 }
 
+func checkGCSFileExists(t *testing.T, bucketName string, fileName string) {
+	ctx := context.Background()
+	client, err := storage.NewClient(ctx)
+	if err != nil {
+		t.Fatalf("storage.NewClient: %v", err)
+	}
+	defer client.Close()
+
+	ctx, cancel := context.WithTimeout(ctx, time.Second*10)
+	defer cancel()
+
+	objAttrs, err := client.Bucket(bucketName).Object(fileName).Attrs(ctx)
+	if err == nil && objAttrs != nil {
+		return
+	}
+	if err != nil {
+		switch err {
+		case storage.ErrObjectNotExist:
+			t.Fatalf("Spritesheet %q does not exist in bucket %q: %v", fileName, bucketName, err)
+		default:
+			t.Fatalf("Error getting bucket attrs: %v", err)
+		}
+	}
```

Comment:
Nit: No need for the `if` and `switch`.

```suggestion
	if err == storage.ErrObjectNotExist {
		t.Fatalf("Spritesheet %q does not exist in bucket %q: %v", fileName, bucketName, err)
	}
	if err != nil {
		t.Fatalf("Error getting bucket attrs: %v", err)
	}
```

Reply:
Thanks!